src/libraries/Common/src/Interop/Windows/NCrypt/Interop.Properties.cs

```diff
@@ -56,7 +56,9 @@ internal static ErrorCode NCryptGetIntProperty(SafeNCryptHandle hObject, string
             {
                 fixed (int* pResult = &result)
                 {
+#if NETSTANDARD || NETCOREAPP
                     Debug.Assert(OperatingSystem.IsWindows());
+#endif
```

Comment:
I don't think `OperatingSystem.IsWindows()` will work on netstandard. But you could use `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` for both cases.

Reply:
Fixed.